distribution/docs/src/main/resources/content/_managing/_running/troubleshooting.adoc

```diff
@@ -79,4 +80,18 @@ This can be caused when another ${branding} is not properly shut down.
 * Verify running java.exe are not ${branding} (e.g., kill/close if necessary).
 * Utilize automated start/stop scripts to run ${branding} as a service.
 
+2+^|*Troubleshooting Certificates*
+
+.6+.^|Connection errors, for example:
```

Comment:
❓Should we say that these will appear in the log files?

Reply:
yes